manifests/dm/dm-cluster.yaml

```diff
@@ -0,0 +1,29 @@
+# IT IS NOT SUITABLE FOR PRODUCTION USE.
+# This YAML describes a basic TiDB cluster with minimum resource requirements,
+# which should be able to run in any Kubernetes cluster with storage support.
+apiVersion: pingcap.com/v1alpha1
+kind: DMCluster
+metadata:
+  name: basic
+  namespace: demo
+spec:
+  version: v2.0.0-beta.2
+  pvReclaimPolicy: Delete
+  discovery:
+    address: "http://basic-discovery.demo.svc:10261"
```

Comment:
Why we need to config the discovery address?

Reply:
Because dmcluster won't start another discovery service for itself. It will just reuse tidb cluster's discovery address.